docs/general-development/working-with-tasks-in-sharepoint-workflows-using-visual-studio-2012.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Working with Tasks in SharePoint Workflows using Visual Studio 2012
-description: Learn about the new and revised workflow task framework that was introduced in SharePoint, which is built on the new Workflow Manager.
-ms.date: 10/15/2022
+title: Working with Tasks in SharePoint 2013 Platform Workflows using Visual Studio
+description: Learn about the revised workflow task framework that was introduced in SharePoint 2013, (and which is available in SharePoint 2016, 2019, and Subscription Edition). This is built on the Workflow Manager (which was more recently superceded by 'SharePoint Workflow Manager').

```

Comment:
Same here... there's no such official product name "SharePoint 2016" or "SharePoint 2019", it's "SharePoint Server 2016, SharePoint Server 2019, SharePoint Server Subscription Edition"